spec/models/report/answer_tally_report_spec.rb

```diff
@@ -151,7 +151,7 @@
       expect(report).to have_data_grid(nil)
     end
 
-    it "counts of options across a select one question and select multiple question" do
+    it "counts of options across a select one question and select multiple question", :investigate do
```

Comment:
Thanks for doing this! We will be keeping a good eye on these moving forward and eventually start fixing them.